.github/scripts/generate-release-body.ts

```diff
@@ -150,12 +150,70 @@ function capitalize(s) {
   return s[0].toUpperCase() + s.slice(1);
 }
 
+// filters out the PR that has a tag `client` or `runtime` and returns all the remaining PRs
+function PRfilter(prLabels: any) {
```

Comment:
Nit-picky general comment - try to keep the style & approach consistent.
The formatting is off compared to other functions and e.g. below where consts are used, you use `let`.